src/angular/styles/_typography.scss

```diff
@@ -228,11 +228,14 @@
   --sbb-fieldset-legend-nested-font-size: #{sbb.pxToRem(20)};
 
   // Forms
-  --sbb-form-background-color-readonly: var(--sbb-color-milk);
-  --sbb-form-border-color-readonly: var(--sbb-color-aluminum);
-  --sbb-form-field-width: #{sbb.pxToRem(320)};
-  --sbb-form-input-color: var(--sbb-color-charcoal);
```

Comment:
`--sbb-form-input-color` is also used by other components. It needs to be changed there, too.